package/MDAnalysis/coordinates/chain.py

```diff
@@ -217,9 +217,11 @@ class ChainReader(base.ProtoReader):
 
     """
     format = 'CHAIN'
+    _n_frames: int
```

Comment:
I'm missing why this is a class attr rather than attribute?

Reply:
this is just a type annotation